core/_forms.scss

```diff
@@ -1,3 +1,6 @@
+$_form-box-shadow: inset 0 1px 3px rgba(#000, 0.06);
```

Comment:
It’d be nice to not have that happen, yeah. Do you want to open a PR for https://github.com/thoughtbot/bitters/issues/260?